lib/edge-runtime/ai.ts

```diff
@@ -102,22 +119,28 @@ export async function Dottie( // Dottie talks to docs
     } as ChatGPTMessage;
 
     chatGptPrompt.push(docMessage);
+    nonSystemMessages.push(docMessage);
     streamInfo(docMessage);
 
-    // If over context limit, remove oldest documentation chunks
+    // If function response too old, remove oldest documentation chunks
     chatGptPrompt = removeOldestFunctionCalls(
       [...chatGptPrompt],
-      model === "gpt-4-0613" ? "4" : "3",
+      undefined,
+      1500, // Cut out old docs retrieved, but keep old questions & answers
```

Comment:
Don't want to fill up context limit of GPT3.5 if we don't have to. The more trash in the context window, the worse it'll perform. In this case, since we're doing embedding search, we hardly care about old function calls at all. So this removes them when they're younger than once they've filled up the whole context window.